platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheIncompatibleTasksIntegrationTest.groovy

```diff
@@ -126,17 +126,16 @@ class ConfigurationCacheIncompatibleTasksIntegrationTest extends AbstractConfigu
         addIncompatibleTasksWithProblems()
 
         when:
-        configurationCacheFails "declared", "notDeclared", "$MAX_PROBLEMS_SYS_PROP=2", WARN_PROBLEMS_CLI_OPT
+        // using warning mode, because otherwise any CC problem results in the main CC build failure, and not too-many-problems failure
```

Comment:
❓ Not sure if I understand the point of the comment. The test already implies that we're using warning mode, otherwise `max-problems` has no effect.

Reply:
Done